sycl/doc/syclcompat/README.md

```diff
@@ -816,19 +837,26 @@ namespace syclcompat {
 
 class device_ext : public sycl::device {
   device_ext();
-  device_ext(const sycl::device &base);
+  device_ext(const sycl::device &base, bool print_on_async_exceptions = false,
+             bool in_order = true);
   ~device_ext();
 
   bool is_native_host_atomic_supported();
-  int get_major_version();
-  int get_minor_version();
-  int get_max_compute_units();
-  int get_max_clock_frequency();
-  int get_integrated();
-  void get_device_info(device_info &out);
+  int get_major_version() const;
+  int get_minor_version() const;
+  int get_max_compute_units() const;
+  int get_max_clock_frequency() const;
+  int get_integrated() const;
+  int get_max_sub_group_size() const;
+  int get_max_register_size_per_work_group() const;
+  int get_max_work_group_size() const;
+  int get_mem_base_addr_align() const;
+  size_t get_global_mem_size() const;
+  void get_memory_info(size_t &free_memory, size_t &total_memory);
```

Comment:
Should this function also be `const` or does this alter the memory within the `device_ext` class?